meta/engineering_principles.md

```diff
@@ -0,0 +1,137 @@
+# Engineering Principles
+
+We have [company values](what_is_artsy.md), and are working on our shared values with design and product. However,
+when we make decisions on what and how to do engineering we need guiding principals. Things that make Artsy's
+development culture unique and drive us to do great work.
+
+### Open Source by Default
+
+We consider the data and relationships that make Artsy difficult to copy, code only represents the past and
+close-to present. A code project should start as open, then move to closed if there are valid business needs that
+need to be hidden.
+
+Working in the open allows individual to retain their work and ideas post-Artsy. We can share the exact problem we
+see with others, and provide great high ways to teach by direct reference.
+
+Expectations for OSS apps are different than OSS libs, an app has little-to-no desire for community contributions.
+Instead it's more of a reference tool. Tools and libraries we can build communities around, but most projects don’t
+get large enough for their stewardship to become burdensome.
+
+- Examples:
+
+  - artsy/meta
+  - artsy/force
+  - artsy/eigen
+  - artsy/emission
+
+- Further Reading:
+
+  - https://artsy.github.io/series/open-source-by-default/
+  - https://ashfurrow.com/blog/open-source-ideology/
+  - https://code.dblock.org/2015/02/09/becoming-open-source-by-default.html
+  - https://www.objc.io/issues/22-scale/artsy/
+
+### Own Your Dependencies
+
+Take the best dependencies to fit your problem and make them better. Rather than try re-invent the wheel, give time
+to take the best dependencies and make them fit into our use cases. We’re not the biggest team, so the most value
+for someone's time can be achieved by writing the glue code between larger projects.
```

Comment:
I like the gist of "we're lean, so try to employ the best components from the wider field," but this feels like it's in conflict with also being core contributors to those "best" dependencies. If they're the best, we should contribute but it's unlikely we'll need to "own" them, right?

Reply:
Great point. 

We use the phrase "own" but it should be more metaphorical I think, for example we're not on the TypeScript core team - but we basically have their digital phone numbers. We probably will never need it, but we can easily make a meeting with them to discuss something critical.

I'll try shift the angle a bit here form a direct "repo access"